dgl_sparse/src/elemenwise_op.cc

```diff
@@ -24,7 +24,10 @@ c10::intrusive_ptr<SparseMatrix> SpSpAdd(
   ElementwiseOpSanityCheck(A, B);
   auto torch_A = COOToTorchCOO(A->COOPtr(), A->value());
   auto torch_B = COOToTorchCOO(B->COOPtr(), B->value());
-  auto sum = (torch_A + torch_B).coalesce();
+  auto sum = torch_A + torch_B;
```

Comment:
This is not a long term solution, let's file a bug to discuss the better solution in the future for avoiding memory copy when converting from DGL sparse to torch sparse.